hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/service/AbstractKeyDeletingService.java

```diff
@@ -658,4 +670,35 @@ public long getMovedFilesCount() {
   public BootstrapStateHandler.Lock getBootstrapStateLock() {
     return lock;
   }
+
+  /**
+   * Submits SetSnapsnapshotPropertyRequest to OM.
+   * @param setSnapshotPropertyRequests request to be sent to OM
+   */
+  protected void submitSetSnapshotRequest(
+      List<OzoneManagerProtocolProtos.SetSnapshotPropertyRequest> setSnapshotPropertyRequests) {
+    if (setSnapshotPropertyRequests.isEmpty()) {
+      return;
+    }
+    OzoneManagerProtocolProtos.OMRequest omRequest = OzoneManagerProtocolProtos.OMRequest.newBuilder()
+        .setCmdType(OzoneManagerProtocolProtos.Type.SetSnapshotProperty)
+        .addAllSetSnapshotPropertyRequests(setSnapshotPropertyRequests)
+        .setClientId(clientId.toString())
+        .build();
+    Map<String, SnapshotInfo> val = new HashMap<>();
+    setSnapshotPropertyRequests
+        .forEach(i -> {
+          try {
+            val.put(i.getSnapshotKey(),
```

Comment:
does the hashmap val get used somewhere?

Reply:
removed it as part of https://github.com/apache/ozone/pull/8450. I was testing something forgot to remove it